test/unit/service/JobRoleServiceTest.ts

```diff
@@ -115,3 +115,77 @@ describe('JobRoleService', function () {
         });
 
 });
+
+
+describe('createJobRole', function () {
+    it('should return response 202 when able to delete job role.', async () => {
+        const id: string = '1';
+        const fakeResponse = {
+            message: 'Job Role with ' + id + ' has been removed!',
+            status: 202,
+        };
+
+        const headers = { 'Authorization': '' };
+
+
+        const mock = new MockAdapter(axios);
+
+        mock.onDelete(jobRoleService.API_URL + '/job-roles/' + id, { headers }).reply(202, fakeResponse);
+
+        const results: any = await jobRoleService.deleteJobRoleById('', id);
+
+        expect(results.message).to.deep.equal(fakeResponse.message);
+        expect(results.status).to.deep.equal(fakeResponse.status);
+    });
+
+    it('should return response 500 when server error occurs.', async () => {
+        const id: string = '1';
+        let error: string;
+
+        const headers = { 'Authorization': '' };
+        const mock = new MockAdapter(axios);
+        mock.onDelete(jobRoleService.API_URL + '/job-roles/' + id, { headers }).reply(500);
+
+        try {
+            await jobRoleService.deleteJobRoleById('', id);
+        } catch (e) {
+            error = e.message;
+        }
+
+        expect(error).to.deep.equal('Could Not Get Job Roles');
+    });
+
+    it('should return response 403 when user is unauthorised to perform action.', async () => {
+        const id: string = '1';
+        let error: string;
+
+        const headers = { 'Authorization': '' };
+        const mock = new MockAdapter(axios);
+        mock.onDelete(jobRoleService.API_URL + '/job-roles/' + id, { headers }).reply(403);
+
+        try {
+            await jobRoleService.deleteJobRoleById('', id);
+        } catch (e) {
+            error = e.message;
+        }
+
+        expect(error).to.deep.equal('You Do Not Have The Correct Permissions');
+    });
+
+    it('should return response 404 when job role id does not exist.', async () => {
```

Comment:
Could you also test that the delete endpoint works without an error (I can help you with this if needed)